src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle {
+    double kat1;
+    double kat2;

```

Comment:
Class fields should be private to adhere to the encapsulation principle. Declare your fields with private access modifiers.